_posts/2025-04-06-cka8762-rit.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+# If your post title is longer or more complicated
+# than can be represented in the filename, uncomment the following line
+# and specify a custom title
+title:  "Improving the RIT Bus app"
+
+Contribution: 
+- Feature Suggestion
+
+# Enter your name below
+author: Cameron Adams
+---
+
+##What I contributed to?
+I contributed to the open source app for rit buses, Ritchie's Bus Schedule. An open source app that focuses on helping RIT students navigate the bus schedule easier to 
+navigate. 
+
+##Why I choose it
+I wanted to contribute to something that I use, even if its every so often. I always found myself looking at the official rit bus site and struggling to read through
+the schedule that they layout. When this class first introduced me to it I swapped over to using [rit-bus.app](rit-bus.app), which was easier to read. I find it more
+satisfying to contribute to an app that not only I use, but hopefully other rit students.
+
+##The Contribution
+Since the app itself, as far as I can tell, runs fairly clean so I didn't immediately find any glaring bugs. I used the app for a couple weeks, doing a combination of 
+limit testing and just general app use, I still found no glaring bugs. There are some differences between the mobile and desktop versions of the websites, but those 
+are intentional experimental or design choices. After taking a look at the issues, I was a little overwhelmed with making code changes.
+
+Instead I decided to make a feature suggestion. While I was using the app, I only have the 1 apex bus favorited. When on desktop I felt like there was a lot of dead space
+in the favorited section that could be used to display more information.
+
+![An image displaying the deadspace in the favorite buses section](https://github.com/KobeAdams/hfoss2025-blogs/blob/main/assets/images/cka8762/Contributions-1.png)
```

Comment:
you dont need to prefix this with the full URL if the files are in the repository

removing the `https://github.com/KobeAdams` and the `blob/main/` from the image URLs should make them show up correctly

```suggestion
![An image displaying the deadspace in the favorite buses section](/hfoss2025-blogs/assets/images/cka8762/Contributions-1.png)
```